joomla/joomla-cms
# regression 3.6.3: Fix xtd buttons close for tinyMCE

Pull Request for Issue # .
### Summary of Changes
- Use jQuery
- This will be reverted in 3.7
### Testing Instructions

Apply patch and make sure that the xtd buttons close when selecting an item (e.g. article, module, etc)
### Documentation Changes Required

None
